src/app/containers/CpsRelatedContent/index.jsx

```diff
@@ -43,38 +42,48 @@ const formatItem = (item, env) => {
   return assocPath(['locators', 'assetUri'], `${baseUri}${uriSuffix}`, item);
 };
 
-const CpsRelatedContent = ({ content }) => {
+const CpsRelatedContent = ({ content, enableGridWrapper }) => {
```

Comment:
Sometimes it's good to be able to infer what type a variable is from its name. A fairly universal standard is to prefix Boolean variable names with "is" or "has", so this might become `isGridWrapperEnabled`. I thought it worth mentioning as there are [quite](https://github.com/bbc/simorgh/blob/569b8b3975a19d9ed915fb8eac10e55cba4fdfd9/cypress/support/config/services.js#L2) [a few](https://github.com/bbc/simorgh/blob/bb3c8c5626d862cb14ee99ae9a89eccdf0a385a3/src/app/containers/Navigation/index.canonical.jsx#L26) [places](https://github.com/bbc/simorgh/blob/e1328fc49879ae47e3031a09b014921c03c562a2/cypress/integration/pages/frontPage/tests.js#L9) [we do this](https://github.com/bbc/simorgh/blob/61dacbb961e63dfde48321b3156dd12c0cbd3928/src/app/containers/ChartbeatAnalytics/canonical/index.jsx#L10).